### PR TITLE
Ports: Add dosfstools port

### PR DESCRIPTION
**Pull request by supercomputer7**
cc @timschumi

### CODE REVIEW

**Review by bgianfo — APPROVED** [review]
LGTM!

File: Ports/dosfstools/package.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='dosfstools'
+version='4.2'
+useconfigure='true'
+use_fresh_config_sub='true'
+config_sub_paths=('config.sub')
+files="https://github.com/dosfstools/dosfstools/archive/refs/tags/v${version}.zip dosfstools-${version}.zip bed48396285facb98e1d323a523654977fb5ddf350597bf4b4de11ae667fd585"
+auth_type='sha256'
+configopts=("--prefix=${DESTDIR}/usr" "--oldincludedir=${DESTDIR}/usr/include/")
```
Comment (timschumi) [inline Ports/dosfstools/package.sh:9]:
```suggestion
configopts=(
    "--prefix=${DESTDIR}/usr"
    "--oldincludedir=${DESTDIR}/usr/include/"
)
```

File: Ports/dosfstools/package.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='dosfstools'
+version='4.2'
+useconfigure='true'
+use_fresh_config_sub='true'
+config_sub_paths=('config.sub')
+files="https://github.com/dosfstools/dosfstools/archive/refs/tags/v${version}.zip dosfstools-${version}.zip bed48396285facb98e1d323a523654977fb5ddf350597bf4b4de11ae667fd585"
```
Comment (timschumi) [inline Ports/dosfstools/package.sh:7]: Why not the actual release tarball? That way we can skip relying on the user-installed `auto*` tools.

File: Ports/dosfstools/package.sh (hunk 1 above)
Comment (timschumi) [inline Ports/dosfstools/package.sh:9]: Why does this appear to install into `/usr`, but we later try to symlink to binaries in `/usr/local`?

**Review by timschumi — CHANGES_REQUESTED** [review]

File: Ports/dosfstools/package.sh (hunk 1 above)
Comment (supercomputer7) [reply Ports/dosfstools/package.sh:9]: It was not needed actually, so I will remove this.

**Pull request merged**